File: Arrays/3516. Find Closest Person/solution.swift
class Solution {
    func findClosest(_ x: Int, _ y: Int, _ z: Int) -> Int {
        
        let first : Int = abs(z-x)
        let second : Int = abs(z - y)
        
        if first < second {
            return 1
        } else if first  == second {
            return 0
        } else {
            return 2
        }
    }
}


let x :Int = 2
let y :Int = 7
let z : Int = 4

print(Solution().findClosest(x, y, z))

File: Arrays/3516. Find Closest Person/solution.py
class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:

        first = abs(z-x)
        second = abs(z-y)

        
        return 1 if first < second else 0 if first == second else 2

if __name__ == '__main__':

    """ Example 1: """
    x = 2
    y = 5
    z = 6
    print(Solution().findClosest(x, y, z))


    """ Example 2: """
    x = 1
    y = 5
    z = 3
    print(Solution().findClosest(x, y, z))